resources/js/Pages/Admin/Dashboard/Index.tsx

```diff
@@ -1,53 +1,116 @@
+import React, { useState } from "react";
 import { AppLayout } from '@/Layouts/AppLayout';
-import React from "react";
-import MentorForm from "@/Organisms/MentorForm";
-
-{
-    /*import React from "react";
-import Heading from "@/Atoms/Heading";
-import Button from "@/Atoms/Button";
-import Card from "@/Molecules/Card";
-import SampleForm from "@/Organisms/SampleForm";
-import AuthLayout from "@/Atoms/AuthLayout";
-import { Textarea } from "@/Atoms/Textarea";
-
-interface Props {
-  message: React.ReactChildren;
-  button: React.ReactChildren;
-}
+import Table  from "@/Molecules/Table";
+import { Member } from '@/Interfaces/Member';
 
-export default function Index({ message, button }: Props) {
-    return (
-        <div>
-            <AuthLayout heading="Auth Layout" message="">
-                <SampleForm />
-                <p>&nbsp;</p>
-                <p>&nbsp;</p>
-
-                <Heading type="h3">{message}</Heading>
-                <Card heading="Card heading" hasPadding={true} />
-                <Button size="medium">{button}</Button>
-                <Textarea placeholder="text area" />
-            </AuthLayout>
-        </div>
-    );
-}
-*/
+type Summary = {
+    mentor:string;
+    mentor_slack_handle:string;
+    mentor_email:string;
+    mentee:string;
+    mentee_slack_handle:string;
+    mentee_email:string;
 }
 
-// interface Props {
-//   message: React.ReactChildren;
-//   button: React.ReactChildren;
-// }
+interface IProps {
+    mentors: Member[];
+    mentees: Member[];
+};
 
-export default function Index() {
+const compare_qty = (a:Member, b:Member):number => {
```

Comment:
the normal naming convention for typescript/javascript functions is to use camelCase - so this would be `compareQuantity`